tsconfig.json

```diff
@@ -25,14 +25,17 @@
     "baseUrl": ".",
     "paths": {
       "@/components/*": [
-        "src/app/components/*"
+        "src/app/[lang]/components/*"
       ],
       "@/lib/*": [
-        "src/app/lib/*"
+        "src/app/[lang]/lib/*"
       ],
       "@/stores/*": [
-        "src/app/stores/*"
-      ]
+        "src/app/[lang]/stores/*"
```

Comment:
I'm not sure that "lib" and "stores" components that aren't directly implied in the display should be moved like that.